flink-table-store-flink/flink-table-store-flink-common/src/main/java/org/apache/flink/table/store/connector/source/TableStoreSource.java

```diff
@@ -209,4 +233,19 @@ public LookupRuntimeProvider getLookupRuntimeProvider(LookupContext context) {
         return TableFunctionProvider.of(
                 new FileStoreLookupFunction(table, projection, joinKey, predicate));
     }
+
+    @Override
+    public TableStats reportStatistics() {
```

Comment:
Can we document this? This may not be a positive path always. Flink SQL provides `table.optimizer.source.report-statistics-enabled` to close this.